app/web/src/components/Navigation/Navigation.tsx

```diff
@@ -127,14 +166,68 @@ const useStyles = makeStyles((theme) => ({
     alignSelf: "center",
     fontWeight: "bold",
   },
+  menu: {
+    boxShadow: theme.shadows[1],
+    minWidth: "12rem",
+  },
+  menuPopover: {
+    transform: "translateY(2rem)",
+  },
+  menuTitle: {
+    fontWeight: "bold",
+    position: "relative",
+    pointerEvents: "none",
+    "&:after": {
+      content: "''",
+      position: "absolute",
+      left: 16,
+      bottom: 0,
+      borderBottom: "1px solid black",
+      width: 32,
+    },
+  },
+  menuItem: {
+    width: "100%",
+  },
+  menuItemMessage: {
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "space-between",
+    alignItems: "center",
+  },
+  notificationCount: {
+    color: grey[500],
+    fontWeight: "bold",
+  },
+  menuBtn: {
+    display: "flex",
+    flexDirection: "row",
+    alignItems: "center",
+    border: `1px solid ${grey[300]}`,
+    borderRadius: "35%",
+    backgroundColor: grey[200],
+    padding: 4,
```

Comment:
The button will look even better with twice as much padding and [a large px border radius](https://stackoverflow.com/questions/29966499/border-radius-in-percentage-and-pixels-px-or-em). Try this:
```suggestion
    borderRadius: 999,
    backgroundColor: grey[200],
    padding: theme.spacing(1), //0.5rem, 8px
```

Reply:
I guess the difference between both us experience plays a role here ^^ look much better yes, thanks

Reply:
That's a nice CSS trick right there!